sharding-core/sharding-core-common/src/main/java/org/apache/shardingsphere/core/metadata/ShardingMetaDataLoader.java

```diff
@@ -30,12 +30,8 @@
 
 import javax.sql.DataSource;
 import java.sql.SQLException;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
star import is not allowed.
you can use `mvn install` to check all code style.

Reply:
Ok, got it, thanks. I fix this problem.